src/main/webapp/app/exercises/modeling/participate/modeling-submission.component.scss

```diff
@@ -13,3 +13,7 @@ jhi-modeling-assessment {
 .editor-large {
     min-height: 100vh;
 }
+
+#problem-statement {
+    font-weight: normal !important;
+}
```

Comment:
You are removing the strong tag from translation too, is it necessary to do the both?